app/services/rabbit.py

```diff
@@ -83,3 +86,4 @@ def listen(self, on_message_callback, queue=None):
             self.channel.stop_consuming()
 
         self.connection.close()
+        return self.consumer_tag
```

Comment:
The consumer tag will only be returned when the application gets stopped manually or after crashing. The consumer tag is irrelevant at that moment. I think the logging should be moved inside this method as I don't see a method to return the consumertag from within the `While True:` loop.

You can see this here where I started the application and it logs the first line, but only after pressing ctrl-c (KeyboardInterupt) it prints the second line with the consumertag:

`python main.py                                                                                                                                                                                                                          
{"message": "Waiting for messages on queue vrt.events.essence...", "logger": "app.app", "level": "info", "timestamp": "2021-02-19T14:10:25.357593Z", "source": "/home/rudolf/stack/viaa/projects/vrt-events-essence/app/app.py:start:497"}
^C{"message": "Consumer tag is: ctag2.63b9f6248220445cbc26129adc27b6fd...", "logger": "app.app", "level": "info", "timestamp": "2021-02-19T14:10:28.213758Z", "source": "/home/rudolf/stack/viaa/projects/vrt-events-essence/app/app.py:start:499"}
`